### PR TITLE
feat: fix style on tables

### LINKED ISSUE #159: properly style tables in preview
A table in an issue description gets correctly converted into an HTML table element, however the style is not as neat as GitHub's. Take [this issue](https://37843ffb.devpool-directory-ui.pages.dev/?issue=2489452377) as an example:

Ours:
image

GitHub's:
image

We should improve this, I believe it's doable with CSS only.

### PR DESCRIPTION
**Pull request by vichydra**
Resolves #159

### CODE REVIEW

**Comment by vichydra** [comment]
Old dark mode:

image

New dark mode:

image



Old light mode:

image


New light mode:

image

**Comment by zugdev** [comment]
Can you please use the same colors as in the code block for the alt color in the table?

**Comment by zugdev** [comment]
Please add bottom-padding to the table, it's not a footer only thing.

image

File: static/style/special.css [hunk 1]
```diff
@@ -1,8 +1,12 @@
 :root {
   --dark-background: #000210;
   --dark-background-half: #00021080;
+  --alt-dark-background: #808080;
```
Comment (zugdev) [inline static/style/special.css:4]: Thanks for the hardcode refac!

**Comment by zugdev** [comment]
Looks good on mobile too:

image

I've noticed the text looks blurry, but it seems to be in prod, I'll fix it.

**Comment by zugdev** [comment]
The blurry text is Opera's fault, it's about how they render text when DevTools is open. Since it only affects when DevTools is open it's not a relevant bug and we should not worry.

**Pull request merged**